### PR TITLE
feat(ReadMe): Improve ReadMe

### PR DESCRIPTION
**Pull request by paradoux**
- Added structure and information to the ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,146 @@
...
+yarn server migrate
+```
 
-## Run migrations
+5. Launch the server
 
 ```sh
-docker-compose run app_server yarn server migrate
+cd packages/server
+yarn start-dev
```
Comment (AlexGeb) [inline README.md:94]: Or simply
```suggestion
yarn server start-dev
```

File: README.md [hunk 2]
```diff
@@ -1,23 +1,146 @@
...
 ```
+
+6. Launch the client
+
+```sh
+cd packages/client
+yarn start-dev
```
Comment (AlexGeb) [inline README.md:101]: Or simply
```suggestion
yarn client start-dev
```

File: README.md [hunk 3]
```diff
@@ -1,23 +1,146 @@
...
+## Contributing
+
+Any contributions you make are **greatly appreciated**, especially styling!
+
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
```
Comment (AlexGeb) [inline README.md:129]: I think you can remove this..

**Review by AlexGeb — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (paradoux) [reply README.md:129]: I thought I would leave it as a guidance for potential other developers that would join the project :))

**Pull request merged**